.jenkins

```diff
@@ -28,6 +28,33 @@ pipeline {
 
         stage('Build') {
             parallel {
+                stage('CUDA-11.4-NVHPC') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.nvhpc'
+                            dir 'scripts/docker'
+                            label 'nvidia-docker && volta && large_images'
+                            args '-v /tmp/ccache.kokkos:/tmp/ccache --env NVIDIA_VISIBLE_DEVICES=$NVIDIA_VISIBLE_DEVICES'
+                        }
+                    }
+                    steps {
+                        sh '''rm -rf build && mkdir -p build && cd build && \
+                              /opt/cmake/bin/cmake \
+                                -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Is `Debug` a conscious choice?

Reply:
No.  You would want to use something else?

Reply:
The build time looked a bit on the big side with around 40 minutes but I think it's fine. We also don't have many `Debug` builds.